src/signals/incident-management/containers/Dashboard/components/Filter/SelectList.tsx

```diff
@@ -78,6 +83,7 @@ const SelectList = ({ filterActiveName, setFilterActiveName }: Props) => {
       filterNames.map((name) => [name, { display: '', value: '' }])
     )
     reset(defaultValues)
+    fetchData('')
```

Comment:
a little strange thats its gets renamed. Maybe rename it in Filter and Select list to fetchData or fetchWithQuerystring to be really specific.

Reply:
great!